docs/developing/installing/targets.rst

```diff
@@ -0,0 +1,3 @@
+.. _Git repo named h: https://github.com/hypothesis/h/
```

Comment:
Admittedly the benefit is slim in this case but I think it might be a good pattern to get into the habit of:

1. De-duplication: if the repo URL changed we'd only have to change it in one place, admittedly it seems unlikely that the repo URL is gonna change.

2. Convenience: it allows you to just write:

        `Git repo named h`_

   each time and have it be linked, instead of having to write

        `Git repo named h <https://github.com/hypothesis/h/>`_

   every time. 

3. Consistency in the source files, even though in this case it doesn't seem likely the URL will change, doing it like this is consistent with the same pattern used for other URLs that we link to multiple times as well and I think it's good to get into the habit / demonstrate best practice.

4. Consistency in the output, since this makes it convenient to write

        `Git repo named h`_

   it increases the chances that whenever we link to the repo it'll always be rendered as
   `<a href="...">Git repo named h</a>` which I think is nice for the reader as they know it's
   the same thing being linked to.